securedrop/tests/test_journalist.py

```diff
@@ -53,8 +53,7 @@ def tearDown(self):
     @patch('crypto_util.CryptoUtil.genrandomid',
            side_effect=['bad', VALID_PASSWORD])
     def test_make_password(self, mocked_pw_gen):
-        class fake_config:
-            pass
+        fake_config = SDConfig()
```

Comment:
please remove this line. If it was necessary it should be **fake_config = SDConfig** (i.e. without the parenthesis). It does not fail because the call does not actually try to use it. But it is a type violation of some kind and the purpose of this PR is precisely to help with type violations so... we don't want that ;-)

Reply:
I will not remove these as the current PR brings in only the limited changes required to have an API level similar thing by replacing `config` module with `SDConfig` class. We can update the tests in a future PR where we update how config works.

Reply:
ok. Then you need to change this to **fake_config = SDConfig** to be exact (i.e. without the parenthesis).